src/app/modules/companies/store/companies.actions.ts

```diff
@@ -0,0 +1,40 @@
+import { PartialDeep } from 'type-fest';
+import { updateOrganization } from './../../../graphql/types/updateOrganization';
+import { EditCompany } from './../../../models/edit-company.model';
+import { ApolloError } from '@apollo/client/core';
+import { createAction, props } from '@ngrx/store';
+import { getOrganization } from 'src/app/graphql/types/getOrganization';
+
+export const getCompany = createAction(
+  '[Companies] Get Company',
+  props<{id: string}>()
+);
+
+export const getCompanySuccess = createAction(
+  '[Companies] Get Company Success',
+  props<{ data: getOrganization }>()
+);
+
+export const getCompanyFailure = createAction(
+  '[Companies] Get Company Failure',
+  props<{ error: ApolloError }>()
+);
+
+export const setCompany = createAction(
+  '[Companies] Set Company',
+  props<{ data: PartialDeep<EditCompany> }>()
+);
```

Comment:
Looks like this action is unused and can be removed

Reply:
It is used. https://github.com/VirtoCommerce/vc-demo-xapi-app/blob/7a8cf4eb057f0bce9580b14916ac64c641421b7d/src/app/modules/companies/components/company-edit/company-edit.component.ts#L68